framework/src/constraints/MortarConstraintBase.C

```diff
@@ -80,11 +80,11 @@ MortarConstraintBase::MortarConstraintBase(const InputParameters & parameters)
              : nullptr),
     _secondary_var(
         isParamValid("secondary_variable")
-            ? _subproblem.getStandardVariable(_tid, parameters.getMooseType("secondary_variable"))
-            : _subproblem.getStandardVariable(_tid, parameters.getMooseType("primary_variable"))),
+            ? _sys.getActualFieldVariable<Real>(_tid, parameters.getMooseType("secondary_variable"))
+            : _sys.getActualFieldVariable<Real>(_tid, parameters.getMooseType("primary_variable"))),
```

Comment:
What's the difference between `getStandardVariable` and `getActualFieldVariable`?

Reply:
`getStandardVariable` returns `MooseVariableFE<Real>`. `getActualFieldVariable` returns `MooseVariableField<Real>` which can be either `MooseVariableFE<Real>` or `MooseVariableFV<Real>`